libcnb-test/src/pack.rs

```diff
@@ -90,3 +92,51 @@ impl From<PackBuildCommand> for Command {
         command
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use std::ffi::OsStr;
+
+    #[test]
+    fn from_pack_build_command_to_command() {
+        let mut input = PackBuildCommand {
+            builder: String::from("builder:20"),
+            path: PathBuf::from("/tmp/foo/bar"),
+            image_name: String::from("my-image"),
+            buildpacks: vec![
+                BuildpackReference::Id(String::from("libcnb/buildpack1")),
+                BuildpackReference::Path(PathBuf::from("/tmp/buildpack2")),
+            ],
+            verbose: true,
+        };
+
+        let command: Command = input.clone().into();
+
+        assert_eq!(command.get_program(), "pack");
+
+        assert_eq!(
+            command.get_args().collect::<Vec<&OsStr>>(),
+            vec![
+                "build",
+                "my-image",
+                "--builder",
+                "builder:20",
+                "--path",
+                "/tmp/foo/bar",
+                "--buildpack",
+                "libcnb/buildpack1",
+                "--buildpack",
+                "/tmp/buildpack2",
+                "-v"
+            ]
+        );
+
+        assert_eq!(command.get_envs().collect::<Vec<_>>(), vec![]);
+
+        // Assert conditional '-v' flag works as expected:
+        input.verbose = false;
+        let command: Command = input.into();
+        assert!(!command.get_args().any(|arg| arg == OsStr::new("-v")));
```

Comment:
A variation on this test: we could use `find` instead of `any` and then `assert_eq!(..., Some())` so if it fails it shows in the error message we were expecting `-v` and didn't get it versus now.